scripts/deploy

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+
+set -e
+
+if [[ -n "${GT_SITE_DEBUG}" ]]; then
+    set -x
+fi
+
+DIR="$(dirname "$0")"
+
+function usage() {
+    echo -n \
+"Usage: $(basename "$0") COMMAND OPTION[S]
+Deploy AWS infrastructure using Terraform.
```

Comment:
I think it would be good to include a note that this script is solely for Travis here.